rules/pkg-normalize.js

```diff
@@ -22,6 +22,10 @@ const isGitRepository = repository => {
 		return false;
 	}
 
+	if (typeof repository !== 'string' && !Array.isArray(repository) && Object.prototype.hasOwnProperty.call(repository, 'url')) {
```

Comment:
I think not but someone may by mistake set the array, for example `repository: ["SamVerschueren/clinton"]` or empty object example `repository: {} this can cause the error.

In our case, the string is obviously correct, an empty object or array is an error. Yes, I would throw an error that the value must be a string. A little later I'll do it.